hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/gas/SystemContractGasCalculatorTest.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.contract.impl.test.exec.gas;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.hedera.hapi.node.transaction.TransactionBody;
+import com.hedera.node.app.service.contract.impl.exec.gas.CanonicalDispatchPrices;
+import com.hedera.node.app.service.contract.impl.exec.gas.SystemContractGasCalculator;
+import com.hedera.node.app.service.contract.impl.exec.gas.TinybarValues;
+import java.util.function.ToLongFunction;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class SystemContractGasCalculatorTest {
+    @Mock
+    private TinybarValues tinybarValues;
+
+    @Mock
+    private ToLongFunction<TransactionBody> feeCalculator;
+
+    @Mock
+    private CanonicalDispatchPrices canonicalDispatchPrices;
+
+    private SystemContractGasCalculator subject;
+
+    @BeforeEach
+    void setUp() {
+        subject = new SystemContractGasCalculator(tinybarValues, canonicalDispatchPrices, feeCalculator);
+    }
+
+    @Test
```

Comment:
For some reason I don't like this test.  It's not wrong: you know and I know that this method`viewGasRequirement` simply returns a constant.  But seeing it right under a `setUp` that creates a `SystemContractGasCalculator` with every single argument mocked is ... I dunno, unit tests are _not_ black box testing so you can expect to know how the unit operates when reading them ... but ... hmmm.  

No suggestion, just whining again.